packages/react-ui/src/components/molecules/Alert.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Alert as MuiAlert, Fade, Box, styled } from '@mui/material';
+import Typography from '../atoms/Typography';
+import AutoAwesomeOutlinedIcon from '@mui/icons-material/AutoAwesomeOutlined';
+
+const StyledAlert = styled(MuiAlert, {
+  shouldForwardProp: (prop) => prop !== 'isNeutral'
+})(({ isNeutral, theme }) => ({
+  borderRadius: theme.spacing(1),
+  paddingLeft: theme.spacing(1.5),
+  alignSelf: 'start',
+  width: '100%',
+  ...(isNeutral
+    ? {
+        backgroundColor: theme.palette.background.default,
+        color: theme.palette.text.primary
+      }
+    : {}),
+  '.MuiAlert-message': {
+    paddingTop: theme.spacing(0.75),
+    paddingBottom: theme.spacing(0.75),
+    width: '100%'
+  },
+  '.MuiAlert-icon': {
+    paddingTop: theme.spacing(1),
+    marginRight: theme.spacing(1),
+    svg: {
+      width: theme.spacing(2),
+      height: theme.spacing(2)
+    }
+  },
+  '.MuiAlert-action': {
+    paddingLeft: theme.spacing(1)
+  }
+}));
```

Comment:
I would add all possible styles directly in the theme, so this way even the Mui Alerts are going to be updated with the new design

Reply:
Some of these rules are linked to the markup changes I made, for example to support actions + close button at the same time. So I'm not sure they will work fine with normal MUI Alerts…